backend/app/services/runtime_work_service.py

```
@@ -0,0 +1,1509 @@
+        package_result = await runtime_rpc_service.call(
+            user_id=user_id,
+            device_id=source.device_id,
+            method="runtime.tasks.prepare_fork_transfer",
+            payload={
+                **source.model_dump(by_alias=True),
+                "transferId": transfer_id,
+            },
+            timeout_seconds=RUNTIME_FORK_TIMEOUT_SECONDS,
+        )
+    except RuntimeRpcError as exc:
+        raise HTTPException(status_code=status.HTTP_502_BAD_GATEWAY, detail=str(exc))
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)